test/catch2/CMakeLists.txt

```diff
@@ -82,40 +86,25 @@ set(unittest_sources
     wrappers/mock_connection.cpp
     wrappers/mock_session.cpp
     ${unittests}
+    ${live_restore_sources}
 )
 
-# Live restore is not supported on Windows
-if(NOT WT_WIN)
```

Comment:
I see we remove window references, what happens to live restore catch2 tests on windows machines now?